spec.md

```diff
@@ -328,6 +328,25 @@ that contains both context and data).
 * Constraints:
   * REQUIRED
 
+### labels
+* Type: `Map`
+* Description: Non-heirarchical attributes which describe the `source` of
```

Comment:
Do we have to limit it to describing the `source` instead of the `event` in general?